### PR TITLE
removed account from custom-repos

### PR DESCRIPTION
**Pull request by tomershinhar**
# Description

removed account from third-party repos routes, services, model and tests.
also dropped account from all other places that are affected  by the change 

Fixes # (issue)

[THEEDGE-2334](https://issues.redhat.com/browse/THEEDGE-2334)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [x] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

File: pkg/services/images_test.go [hunk 1]
```diff
@@ -518,72 +518,40 @@ var _ = Describe("Image Service Test", func() {
 		Context("when creating an image using third party repository", func() {
 			It("should validate the images with empty repos", func() {
 				var repos []models.ThirdPartyRepo
-				account := "00000"
 				orgID := "11111"
-				err := services.ValidateAllImageReposAreFromAccountOrOrgID(account, orgID, repos)
+				err := services.ValidateAllImageReposAreFromAccountOrOrgID(orgID, repos)
```
Comment (mgold1234) [inline pkg/services/images_test.go:522]: nead to change the name of function as well - without account

File: pkg/services/images_test.go [hunk 2]
```diff
@@ -518,72 +518,40 @@ var _ = Describe("Image Service Test", func() {
...
 				orgID := "00000"
-				account := ""
 				repo1 := models.ThirdPartyRepo{OrgID: orgID, Name: faker.UUIDHyphenated(), URL: "https://repo1.simple.com"}
 				result := db.DB.Create(&repo1)
 				Expect(result.Error).ToNot(HaveOccurred())
 				repo2 := models.ThirdPartyRepo{OrgID: orgID, Name: faker.UUIDHyphenated(), URL: "https://repo2.simple.com"}
 				result = db.DB.Create(&repo2)
 				Expect(result.Error).ToNot(HaveOccurred())
-				err := services.ValidateAllImageReposAreFromAccountOrOrgID(account, orgID, []models.ThirdPartyRepo{repo1, repo2})
+				err := services.ValidateAllImageReposAreFromAccountOrOrgID(orgID, []models.ThirdPartyRepo{repo1, repo2})
```
Comment (mgold1234) [inline pkg/services/images_test.go:541]: same

File: pkg/services/images_test.go [hunk 3]
```diff
@@ -518,72 +518,40 @@ var _ = Describe("Image Service Test", func() {
...
 				orgID2 := "2222222"
-				account := ""
 				repo1 := models.ThirdPartyRepo{OrgID: orgID1, Name: faker.UUIDHyphenated(), URL: "https://repo1.simple.com"}
 				result := db.DB.Create(&repo1)
 				Expect(result.Error).ToNot(HaveOccurred())
 				repo2 := models.ThirdPartyRepo{OrgID: orgID2, Name: faker.UUIDHyphenated(), URL: "https://repo2.simple.com"}
 				result = db.DB.Create(&repo2)
 				Expect(result.Error).ToNot(HaveOccurred())
-				err := services.ValidateAllImageReposAreFromAccountOrOrgID(account, orgID1, []models.ThirdPartyRepo{repo1, repo2})
+				err := services.ValidateAllImageReposAreFromAccountOrOrgID(orgID1, []models.ThirdPartyRepo{repo1, repo2})
```
Comment (mgold1234) [inline pkg/services/images_test.go:554]: same

File: pkg/services/images_test.go (hunk 1 above)
Comment (tomershinhar) [reply pkg/services/images_test.go]: done

**Review by ldjebran — COMMENTED** [review]
seems OK one comment

File: unleash/features/feature.go [hunk 4]
```diff
@@ -11,10 +11,10 @@ const FeatureCustomRepos = "fleet-management.custom-repos"
 //FeatureImageBuildMS is the const of the ms build feature flag
 const FeatureImageBuildMS = "fleet-management.images_iso"
 
-// CheckFeatureWithAccount checks to see if a given feature is available for a given account
-func CheckFeatureWithAccount(account string, feature string) bool {
+// CheckFeatureWithOrgID checks to see if a given feature is available for a given orgID
+func CheckFeatureWithOrgID(orgID string, feature string) bool {
 	unleashCtx := unleashCTX.Context{
-		UserId: account,
+		UserId: orgID,
```
[the remark this answers is not in the document]
Comment (AaronH88) [reply unleash/features/feature.go:17]: We can use the new one that no longer needs account: https://github.com/RedHatInsights/edge-api/blob/main/unleash/features/feature.go#L23

[the remark this answers is not in the document]
Comment (tomershinhar) [reply unleash/features/feature.go:17]: @AaronH88 should i just remove https://github.com/RedHatInsights/edge-api/blob/main/unleash/features/feature.go#L15 then?

**Comment by mgold1234** [comment]
@tomershinhar can you fix the conflict please?

**Comment by tomershinhar** [comment]
@mgold1234 just did (:

**Review by AaronH88 — APPROVED** [review]
/lgtm

**Pull request merged**